src/test/js/multibranch/testResults.js

```diff
@@ -40,8 +40,11 @@ module.exports = {
         blueRunDetailsPage.clickTab('tests');
 
         // Expand the test.
-        browser.useXpath().click('//div[@class="result-item-head"]');
-
-        browser.useXpath().waitForElementVisible('//div[@class="test-console"]/h4')
+        browser.useXpath().waitForElementVisible('//span[contains(text(), "StdOut")]');
```

Comment:
as long as the assertions happen in testResults.js - I have had a nightmare of a time this week when stuff fails and it isn't directly in the test code. Takes a long time to track down. Even having the paths elsewhere is a pain as it just says @thingie failed to be found, which then takes a while to drill into when looking at a log.